client/me/help/help-contact-form/index.jsx

```diff
@@ -38,6 +36,7 @@ export const HelpContactForm = React.createClass( {
 		showSubjectField: PropTypes.bool,
 		showSiteField: PropTypes.bool,
 		showHelpLanguagePrompt: PropTypes.bool,
+		selectedSite: PropTypes.object,
```

Comment:
I was thinking that if we ever need any other site details in the future they'll be there, I guess I could just send the id for now and we could rework it later.